config/src/main/java/org/hyperledger/besu/config/JsonUtil.java

```diff
@@ -466,4 +511,94 @@ private static boolean validateInt(final JsonNode node) {
     }
     return true;
   }
+
+  /**
+   * Get the JSON representation of a genesis file without a specific field.
+   *
+   * @param genesisFile The genesis file to read.
+   * @param excludedFieldName The field to exclude from the JSON representation.
+   * @return The JSON representation of the genesis file without the excluded field.
+   */
+  public static String getJsonFromFileWithout(
+      final File genesisFile, final String excludedFieldName) {
+    StringBuilder jsonBuilder = new StringBuilder();
+    JsonFactory jsonFactory =
+        JsonFactory.builder()
+            .configure(JsonFactory.Feature.INTERN_FIELD_NAMES, false)
+            .configure(JsonFactory.Feature.CANONICALIZE_FIELD_NAMES, false)
+            .build();
+    try (JsonParser parser = jsonFactory.createParser(genesisFile)) {
+      JsonToken token;
+      while ((token = parser.nextToken()) != null) {
+        if (token == JsonToken.START_OBJECT) {
+          jsonBuilder.append(handleObject(parser, excludedFieldName));
+        }
+      }
+    } catch (Exception e) {
+      throw new RuntimeException(e);
+    }
+    return jsonBuilder.toString();
+  }
```

Comment:
Have you checked if Jackson natively support excluding fields without having to implement the parsing methods?

Reply:
Yes, I have checked it, and this is the fastest method with the least memory usage. I will check it again in a moment.